cmd/syft/internal/options/license.go

```diff
@@ -14,15 +16,17 @@ var _ interface {
 } = (*licenseConfig)(nil)
 
 func (o *licenseConfig) DescribeFields(descriptions clio.FieldDescriptionSet) {
-	descriptions.Add(&o.IncludeUnknownLicenseContent, `include the content of a license in the SBOM when syft
+	descriptions.Add(&o.IncludeUnknownLicenseContent, `deprecated: include the content of a license in the SBOM when syft
```

Comment:
ideally we could remove the description for this so it doesn't show up in the config command output, but I don't think we have a way (yet) to tell clio/fangs to hide this field. It might be that we do with the right combination of `yaml` and `mapstructure` tag values (ignore yaml specify mapstructure), but I haven't tried this.

Reply:
I'll take a look to see what this could look like

Reply:
I could not get this to work with a couple different combinations. I think it's something we might have to add to the clio library